ae-spendtx-voting-count/ae-vote-count-1.js

```diff
@@ -69,29 +69,35 @@ const countVotes = async () => {
     // 3. check stake for each voter account at height
     console.log("3. check stake for each voter account at height");
     const client = await getClient();
-    const votingAccountStakes = await Promise.all(votingAccounts.map(async (vote) => {
+    const votingAccountStakes = [];
+    for (let vote of votingAccounts) {
```

Comment:
not sure if this is optimized. With `Promise.all` the code runs in parallel while this is a sequential approach

Reply:
Yes, this did lead to a bug, as with to many concurrent requests the API answers with 503 temporary overload after ~100 simultaneous requests, in that case we did assume 0 stake. So now it's slower but always works, if new errors pop up they are clearly logged.